scripts/controls.js

```diff
@@ -22,17 +22,28 @@ export default class Controls {
     if (!tokenButton) return;
 
     tokenButton.tools.push(this._convenientEffectsAppButton);
-    tokenButton.tools.push(this._removeEffectsButton);
+    if (this._settings.unifiedAppButton === 'none') tokenButton.tools.push(this._removeEffectsButton);
   }
 
   get _convenientEffectsAppButton() {
+    const title = this._settings.unifiedAppButton !== 'none' && game.user.role >= this._settings.removeControlsPermission
```

Comment:
This check `this._settings.unifiedAppButton !== 'none' && game.user.role >= this._settings.removeControlsPermission` could probably be extracted to a little helper function for readability since you do it twice

Reply:
Added 
```js
 get _unifiedButton() {
    return this._settings.unifiedAppButton;
  }

  get _userAppControlsPermission() {
    return game.user.role >= this._settings.appControlsPermission;
  }

  get _userRemoveControlsPermission() {
    return game.user.role >= this._settings.removeControlsPermission;
  }
  ```